src/lib/utils/textFile.ts

```diff
@@ -0,0 +1,10 @@
+export async function fetchTextFile(textFileUrl: string, useCache = true) {
+  const response = await fetch(
+    textFileUrl,
+    useCache ? {} : { cache: "no-cache" }
+  );
+  if (!response.ok) {
+    throw `Error when fetching ${textFileUrl}: ${response.status} ${response.statusText}`;
```

Comment:
maybe we can catch it here instead (question above)

Reply:
sentry capture added